diff -rupN a/ntv2sdklinux_12.4.2.1/ntv2projects/classes/ntv2utils.h b/ntv2sdklinux_12.4.2.1/ntv2projects/classes/ntv2utils.h
--- a/ntv2sdklinux_12.4.2.1/ntv2projects/classes/ntv2utils.h	2016-10-29 13:46:43.784790745 +0200
+++ b/ntv2sdklinux_12.4.2.1/ntv2projects/classes/ntv2utils.h	2016-10-29 14:37:25.756878939 +0200
@@ -631,7 +631,8 @@ typedef struct NTV2FormatDescriptor
 	ULWord	linePitch;			///< @brief	Number of 32-bit words per line
 	ULWord	firstActiveLine;	///< @brief	First active line of video (0 if vanc not enabled)
 
-	explicit inline	NTV2FormatDescriptor ()	: numLines (0), numPixels (0), linePitch (0), firstActiveLine (0)	{}	///< @brief	My default constructor
+	/** gcc-6 does not allow to call explicit default constructor calls in array initializations (see ntv2util.cpp:213 */
+	inline	NTV2FormatDescriptor ()	: numLines (0), numPixels (0), linePitch (0), firstActiveLine (0)	{}	///< @brief	My default constructor
 
 	/**
 		@brief	Construct from line and pixel count, plus line pitch.
