diff -rupN a/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/Makefile b/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/Makefile
--- a/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/Makefile	2016-10-28 10:10:47.888000000 +0000
+++ b/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/Makefile	2016-10-28 09:59:37.824000000 +0000
@@ -12,7 +12,7 @@ TARGET 	 := ntv2watcher
 
 QT_PRO_FILE = $(TARGET).pro
 QMAKE_FILE = qMakefile
-QMAKE ?= qmake
+QMAKE ?= qmake-qt5
 
 .PHONY: clean
 
 
diff -rupN a/ntv2sdklinux_12.4.2.1/ntv2projects/qt.mk b/ntv2sdklinux_12.4.2.1/ntv2projects/qt.mk
--- a/ntv2sdklinux_12.4.2.1/ntv2projects/qt.mk	2016-10-28 10:10:47.896000000 +0000
+++ b/ntv2sdklinux_12.4.2.1/ntv2projects/qt.mk	2016-10-28 10:01:57.472000000 +0000
@@ -10,7 +10,7 @@
 # so make sure that we do that.
 
 # WTF?  qmake -v outputs to stderr??
-QMAKE_V := $(shell qmake -v 2>&1)
+QMAKE_V := $(shell qmake-qt5 -v 2>&1)
 QMAKE_4 := $(findstring 4.,$(QMAKE_V))
 QMAKE_5 := $(findstring 5.,$(QMAKE_V))
 
