diff -rupN a/EpochLinuxDriver_V5_11_0_19/applications/Makefile b/EpochLinuxDriver_V5_11_0_19/applications/Makefile
--- a/EpochLinuxDriver_V5_11_0_19/applications/Makefile	2016-10-31 10:41:08.496766802 +0100
+++ b/EpochLinuxDriver_V5_11_0_19/applications/Makefile	2016-10-31 10:44:38.000774517 +0100
@@ -2,7 +2,7 @@ OBJS = $(TARGET).o
 CC = g++
 SDK_DIR =../release/driver
 
-LINUX_KERNEL_VERSION := $(shell uname -r)
+LINUX_KERNEL_VERSION := $(shell ls -1 /lib/modules | sort | head -n 1)
 KERNELDIR := /lib/modules/$(LINUX_KERNEL_VERSION)/build
 BLUE_MACHINE_ARCH=$(shell uname -m)
 
diff -rupN a/EpochLinuxDriver_V5_11_0_19/drivers/orac/Makefile b/EpochLinuxDriver_V5_11_0_19/drivers/orac/Makefile
--- a/EpochLinuxDriver_V5_11_0_19/drivers/orac/Makefile	2016-10-31 10:41:08.676766808 +0100
+++ b/EpochLinuxDriver_V5_11_0_19/drivers/orac/Makefile	2016-10-31 10:44:38.012774518 +0100
@@ -35,7 +35,7 @@ obj ?= .
 
 PWD := $(shell pwd)
 MODULE_OBJECT := $(MODULE_NAME).ko
-KERNEL_UNAME   := $(shell uname -r)
+KERNEL_UNAME   := $(shell ls -1 /lib/modules | sort | head -n 1)
 KERNEL_SOURCES := /lib/modules/$(KERNEL_UNAME)/build/
 
 EXTRA_CFLAGS += -D_LOOSE_KERNEL_NAMES -D__KERNEL__ -DMODULE  -DNTRM -D_GNU_SOURCE -D_LOOSE_KERNEL_NAMES -D__KERNEL__ -DMODULE -DUSE_NEW_VIDEO_FIFO
diff -rupN a/EpochLinuxDriver_V5_11_0_19/samples/CaptureAncillary/ReadClosedCaptions/Makefile b/EpochLinuxDriver_V5_11_0_19/samples/CaptureAncillary/ReadClosedCaptions/Makefile
--- a/EpochLinuxDriver_V5_11_0_19/samples/CaptureAncillary/ReadClosedCaptions/Makefile	2016-10-31 10:41:08.500766802 +0100
+++ b/EpochLinuxDriver_V5_11_0_19/samples/CaptureAncillary/ReadClosedCaptions/Makefile	2016-10-31 10:44:38.000774517 +0100
@@ -2,7 +2,7 @@ OBJS = $(TARGET).o
 CC = g++
 SDK_DIR =../../../release/driver
 
-LINUX_KERNEL_VERSION := $(shell uname -r)
+LINUX_KERNEL_VERSION := $(shell ls -1 /lib/modules | sort | head -n 1)
 KERNELDIR := /lib/modules/$(LINUX_KERNEL_VERSION)/build
 BLUE_MACHINE_ARCH=$(shell uname -m)
 
diff -rupN a/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoDevices/Makefile b/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoDevices/Makefile
--- a/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoDevices/Makefile	2016-10-31 10:41:08.536766802 +0100
+++ b/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoDevices/Makefile	2016-10-31 10:44:38.008774518 +0100
@@ -1,7 +1,7 @@
 OBJS = $(TARGET).o
 CC = gcc
 
-LINUX_KERNEL_VERSION := $(shell uname -r)
+LINUX_KERNEL_VERSION := $(shell ls -1 /lib/modules | sort | head -n 1)
 KERNELDIR := /lib/modules/$(LINUX_KERNEL_VERSION)/build
 BLUE_MACHINE_ARCH=$(shell uname -m)
 
diff -rupN a/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoModes/Makefile b/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoModes/Makefile
--- a/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoModes/Makefile	2016-10-31 10:41:08.536766802 +0100
+++ b/EpochLinuxDriver_V5_11_0_19/samples/CaptureVideo/V4L2/EnumVideoModes/Makefile	2016-10-31 10:44:38.008774518 +0100
@@ -1,7 +1,7 @@
 OBJS = $(TARGET).o
 CC = gcc
 
-LINUX_KERNEL_VERSION := $(shell uname -r)
+LINUX_KERNEL_VERSION := $(shell ls -1 /lib/modules | sort | head -n 1)
 KERNELDIR := /lib/modules/$(LINUX_KERNEL_VERSION)/build
 BLUE_MACHINE_ARCH=$(shell uname -m)
 
diff -rupN a/EpochLinuxDriver_V5_11_0_19/samples/Duplex/Makefile b/EpochLinuxDriver_V5_11_0_19/samples/Duplex/Makefile
--- a/EpochLinuxDriver_V5_11_0_19/samples/Duplex/Makefile	2016-10-31 10:41:08.500766802 +0100
+++ b/EpochLinuxDriver_V5_11_0_19/samples/Duplex/Makefile	2016-10-31 10:44:38.004774517 +0100
@@ -2,7 +2,7 @@ OBJS = $(TARGET).o
 CC = g++
 SDK_DIR=../../release/driver
 
-LINUX_KERNEL_VERSION := $(shell uname -r)
+LINUX_KERNEL_VERSION := $(shell ls -1 /lib/modules | sort | head -n 1)
 KERNELDIR := /lib/modules/$(LINUX_KERNEL_VERSION)/build
 BLUE_MACHINE_ARCH=$(shell uname -m)
 
diff -rupN a/EpochLinuxDriver_V5_11_0_19/samples/misc/Makefile b/EpochLinuxDriver_V5_11_0_19/samples/misc/Makefile
--- a/EpochLinuxDriver_V5_11_0_19/samples/misc/Makefile	2016-10-31 10:41:08.536766802 +0100
+++ b/EpochLinuxDriver_V5_11_0_19/samples/misc/Makefile	2016-10-31 10:44:38.008774518 +0100
@@ -2,7 +2,7 @@ OBJS = $(TARGET).o
 CC = g++
 SDK_DIR =../../release/driver
 
-LINUX_KERNEL_VERSION := $(shell uname -r)
+LINUX_KERNEL_VERSION := $(shell ls -1 /lib/modules | sort | head -n 1)
 KERNELDIR := /lib/modules/$(LINUX_KERNEL_VERSION)/build
 BLUE_MACHINE_ARCH=$(shell uname -m)
 
