diff -rupN a/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracAlsa.c b/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracAlsa.c
--- a/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracAlsa.c	2018-06-08 09:17:24.545170284 +0200
+++ b/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracAlsa.c	2018-06-08 09:20:01.031886189 +0200
@@ -441,7 +441,7 @@ static struct snd_pcm_ops orac_alsa_capt
 	.prepare =		orac_alsa_pcm_capture_prepare,
 	.trigger =		orac_alsa_pcm_capture_trigger,
 	.pointer =		orac_alsa_pcm_capture_pointer,
-#if LINUX_VERSION_CODE < KERNEL_VERSION(4,13,0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,9,2)
 	.silence =      orac_alsa_pcm_capture_silence,
 #else
 	.fill_silence = orac_alsa_pcm_capture_silence,
