diff -rupN a/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracDMABufferMap.c b/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracDMABufferMap.c
--- a/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracDMABufferMap.c	2017-02-11 13:39:17.390475532 +0100
+++ b/EpochLinuxDriver_V5_11_0_26/drivers/orac/OracDMABufferMap.c	2017-02-11 15:36:01.748721014 +0100
@@ -1,5 +1,6 @@
 #include "OracDriver.h"
 #include <linux/pagemap.h>
+#include <linux/version.h>
 #include "OracDMAUtil.h"
 #include <linux/time.h>
 #define DMA_DEBUG 
@@ -252,7 +253,11 @@ void bluedma_unmap_userbuffer(struct blu
 	pci_unmap_sg(devExt->pci_dev_p, dma_buffer->pOSScatterGatherList, pagesMapped, (dma_buffer->dmaDirection == DMA_WRITE)?PCI_DMA_TODEVICE:PCI_DMA_FROMDEVICE);
 
 	for(i=0; i < dma_buffer->nr_pages; i++)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
+		page_cache_release(dma_buffer->pages[i]);
+#else
 		put_page(dma_buffer->pages[i]);
+#endif
 
 	blue_clear_User_DMA_Request(&dma_buffer->userIrp);
 	dma_buffer->buf = NULL;
