--- a/debian/rules	2017-11-24 10:58:10.000000000 +0100
+++ b/debian/rules	2017-12-02 02:03:09.291676525 +0100
@@ -66,7 +66,7 @@ override_dh_auto_configure:
 	dh_auto_configure -- --disable-profile --libdir=$(LIBDIR) --disable-debug --enable-ipv6 --enable-plugins \
 		--enable-sdl --enable-gl --enable-rtdxt \
 		--enable-portaudio --disable-jack-transport --enable-jack \
-		--enable-alsa --enable-scale --enable-qt --disable-quicktime \
+		--enable-alsa --enable-scale --disable-qt --disable-quicktime \
 		--disable-coreaudio --disable-sage --enable-screen\
 		--enable-v4l2 --enable-gpl-build --enable-libavcodec --enable-scale --enable-uyvy \
 		--disable-rtsp \
