diff -rupN a/ntv2sdklinux_13.1.0.1/Makefile b/ntv2sdklinux_13.1.0.1/Makefile
--- a/ntv2sdklinux_13.1.0.1/Makefile
+++ b/ntv2sdklinux_13.1.0.1/Makefile
@@ -18,7 +18,7 @@ ifndef AJA_NO_FLTK
 	$(call make_if_dir_exists, "$(A_APPS_XPLAT_PATH)/watcher")
 	$(call make_if_dir_exists, "$(A_APPS_XPLAT_PATH)/xenacablesfltk")
 endif
-	$(call make_if_dir_exists, $(A_DEMOAPPS_PATH))
+	#$(call make_if_dir_exists, $(A_DEMOAPPS_PATH))
 
 .PHONY:	clean
 
@@ -32,5 +32,5 @@ ifndef AJA_NO_FLTK
 	$(call clean_if_dir_exists, "$(A_APPS_XPLAT_PATH)/watcher")
 	$(call clean_if_dir_exists, "$(A_APPS_XPLAT_PATH)/xenacablesfltk")
 endif
-	$(call clean_if_dir_exists, $(A_DEMOAPPS_PATH))
+	#$(call clean_if_dir_exists, $(A_DEMOAPPS_PATH))
 
