diff -rupN a/VideoMasterHD_SDK_Linux_v6.04/Drivers/DVI/USBufMngr_Linux.c b/VideoMasterHD_SDK_Linux_v6.04/Drivers/DVI/USBufMngr_Linux.c
--- a/VideoMasterHD_SDK_Linux_v6.04/Drivers/DVI/USBufMngr_Linux.c
+++ b/VideoMasterHD_SDK_Linux_v6.04/Drivers/DVI/USBufMngr_Linux.c
@@ -17,7 +17,7 @@
 #include "OSDrvData.h"
 #include <linux/pagemap.h>
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,6,0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,4,60)
 #define page_cache_release(page)        put_page(page)
 #endif
 
@@ -66,7 +66,7 @@ BOOL32 USBufMngr_RetrievePages( PDEVICE_
 
          down_read(&current->mm->mmap_sem);
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,4,60)
          RetVal_i = get_user_pages(current, current->mm, ((unsigned long) pUSBufAddr), NbPages_UL, 1, 0, ppPage_X, NULL);
 #elif LINUX_VERSION_CODE < KERNEL_VERSION(4,9,0)
          RetVal_i = get_user_pages(((unsigned long) pUSBufAddr), NbPages_UL, 1, 0, ppPage_X, NULL);
diff -rupN a/VideoMasterHD_SDK_Linux_v6.04/Drivers/HD/USBufMngr_Linux.c b/VideoMasterHD_SDK_Linux_v6.04/Drivers/HD/USBufMngr_Linux.c
--- a/VideoMasterHD_SDK_Linux_v6.04/Drivers/HD/USBufMngr_Linux.c
+++ b/VideoMasterHD_SDK_Linux_v6.04/Drivers/HD/USBufMngr_Linux.c
@@ -17,7 +17,7 @@
 #include "OSDrvData.h"
 #include <linux/pagemap.h>
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,6,0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,4,60)
 #define page_cache_release(page)        put_page(page)
 #endif
 
@@ -66,7 +66,7 @@ BOOL32 USBufMngr_RetrievePages( PDEVICE_
 
          down_read(&current->mm->mmap_sem);
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,4,60)
          RetVal_i = get_user_pages(current, current->mm, ((unsigned long) pUSBufAddr), NbPages_UL, 1, 0, ppPage_X, NULL);
 #elif LINUX_VERSION_CODE < KERNEL_VERSION(4,9,0)
          RetVal_i = get_user_pages(((unsigned long) pUSBufAddr), NbPages_UL, 1, 0, ppPage_X, NULL);
diff -rupN a/VideoMasterHD_SDK_Linux_v6.04/Drivers/X300/USBufMngr_Linux.c b/VideoMasterHD_SDK_Linux_v6.04/Drivers/X300/USBufMngr_Linux.c
--- a/VideoMasterHD_SDK_Linux_v6.04/Drivers/X300/USBufMngr_Linux.c
+++ b/VideoMasterHD_SDK_Linux_v6.04/Drivers/X300/USBufMngr_Linux.c
@@ -17,7 +17,7 @@
 #include "OSDrvData.h"
 #include <linux/pagemap.h>
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,6,0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,4,60)
 #define page_cache_release(page)        put_page(page)
 #endif
 
@@ -66,7 +66,7 @@ BOOL32 USBufMngr_RetrievePages( PDEVICE_
 
          down_read(&current->mm->mmap_sem);
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,4,60)
          RetVal_i = get_user_pages(current, current->mm, ((unsigned long) pUSBufAddr), NbPages_UL, 1, 0, ppPage_X, NULL);
 #elif LINUX_VERSION_CODE < KERNEL_VERSION(4,9,0)
          RetVal_i = get_user_pages(((unsigned long) pUSBufAddr), NbPages_UL, 1, 0, ppPage_X, NULL);
