diff -rupN a/Bluefish_linux_driver_6_0_1_21/apis/BlueVelvetC/lin/Makefile b/Bluefish_linux_driver_6_0_1_21/apis/BlueVelvetC/lin/Makefile
--- a/Bluefish_linux_driver_6_0_1_21/apis/BlueVelvetC/lin/Makefile	2020-01-16 11:33:28.306884566 +0100
+++ b/Bluefish_linux_driver_6_0_1_21/apis/BlueVelvetC/lin/Makefile	2020-01-16 12:03:40.421622774 +0100
@@ -6,6 +6,8 @@ BLUE_LIB_DIR =../../../build/lib
 CXX=g++
 SRC = ../BlueVelvetC.cpp ../BlueVelvetCImpl.cpp ../BlueVelvetCHancUtils.cpp  BlueVelvetCImpl_Lin.cpp ../BlueVelvetCUtils.cpp ../BlueUtils.cpp
 CXXFLAGS =  -m64 -pipe -std=c++11 -O2 -Wall -W -D_REENTRANT -fPIC -g -I/usr/include -I. -I$(BLUE_INCLUDE_DIR) -DBLUEVELVETC_EXPORTS 
+DESTDIR ?= /
+libdir ?= /usr/lib
 
 # Ignore some warnings!
 CXXFLAGS += -Wno-unused-parameter
@@ -14,15 +16,9 @@ BLUE_MACHINE_ARCH=$(shell uname -m)
 
 ifeq ($(BLUE_MACHINE_ARCH) ,x86_64)
 TARGET=libBlueVelvetC64.so
-ifeq ($(shell if test -d "/usr/lib64/"; then echo yes; fi),yes)
-SYSTEM_LIB_DIR=lib64
-else
-SYSTEM_LIB_DIR=lib
-endif
 CFLAGS += -m64
 else
 TARGET=libBlueVelvetC32.so
-SYSTEM_LIB_DIR=lib
 endif
 OBJS = $(TARGET)
 
@@ -34,12 +30,14 @@ $(SRC:.cpp=.o):
 $(TARGET):	$(SRC:.cpp=.o)
 		$(CXX) -g -shared -Wl,-soname,$(TARGET) -o $(TARGET) $^ -lc -rdynamic 
 install: all
+		mkdir -p $(DESTDIR)$(libdir)
+		-cp -f $(TARGET) $(DESTDIR)$(libdir)/$(TARGET)
 		-cp ../BlueVelvetC.h $(BLUE_INCLUDE_DIR)/
 		-cp ../BlueVelvetCUtils.h $(BLUE_INCLUDE_DIR)/
 		-cp ../BlueVelvetCFuncPtr.h $(BLUE_INCLUDE_DIR)/
 		mkdir -p $(BLUE_LIB_DIR)
 		-cp $(TARGET) $(BLUE_LIB_DIR)/
-		-cp -f $(TARGET) /usr/$(SYSTEM_LIB_DIR)/$(TARGET)
+		-cp $(TARGET) $(DESTDIR)$(libdir)/
 
 clean:
 	rm -f *.o
