diff -rupN a/EpochLinuxDriver_V5_11_0_21/drivers/orac/OracDMABufferMap.c b/EpochLinuxDriver_V5_11_0_21/drivers/orac/OracDMABufferMap.c
--- a/EpochLinuxDriver_V5_11_0_21/drivers/orac/OracDMABufferMap.c
+++ b/EpochLinuxDriver_V5_11_0_21/drivers/orac/OracDMABufferMap.c
@@ -258,7 +258,7 @@ void bluedma_unmap_userbuffer(struct blu
 			(dma_buffer->dmaDirection == DMA_WRITE)?PCI_DMA_TODEVICE:PCI_DMA_FROMDEVICE);
 
 	for (i=0; i < dma_buffer->nr_pages; i++)
-#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,4,60)
 		page_cache_release(dma_buffer->pages[i]);
 #else
 		put_page(dma_buffer->pages[i]);
