--- scripts/install_steps.orig  2022-03-25 08:31:08.039585295 +0100
+++ scripts/install_steps       2022-03-25 08:35:55.767545116 +0100
@@ -378,11 +378,11 @@
 ldconfig

 echo Installing XIMEA-GenTL library
-cp libs/gentl/X$platform_bits/libXIMEA_GenTL.cti $optpath/lib/
+#cp libs/gentl/X$platform_bits/libXIMEA_GenTL.cti $optpath/lib/
 Check

 echo Installing xiFapi-GenTL library
-cp libs/gentl/X$platform_bits/libxiFapi_GenTL.cti $optpath/lib/
+#cp libs/gentl/X$platform_bits/libxiFapi_GenTL.cti $optpath/lib/
 Check

 if [ -e libs/xiapi_dng_store/X$platform_bits ]
