diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules	2017-12-15 17:14:35.118354118 +0100
+++ b/ultragrid/debian.rules	2017-12-15 17:14:36.278375875 +0100
@@ -21,7 +21,7 @@ CARDCONF_BLUEFISH=--enable-bluefish444 -
 #####################################################
 # > dvs
 #####################################################
-CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
+#CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl	2017-12-15 17:14:35.118354118 +0100
+++ b/ultragrid/ultragrid.spec.tpl	2017-12-15 17:14:36.238375126 +0100
@@ -109,7 +109,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > dvs
 #####################################################
-%define build_dvs 1
+#%%define build_dvs 1
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules	2017-12-15 17:14:35.114354044 +0100
+++ b/ultragrid-nightly/debian.rules	2017-12-15 17:14:36.214374675 +0100
@@ -21,7 +21,7 @@ CARDCONF_BLUEFISH=--enable-bluefish444 -
 #####################################################
 # > dvs
 #####################################################
-CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
+#CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-12-15 17:14:35.114354044 +0100
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-12-15 17:14:36.106372650 +0100
@@ -125,7 +125,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > dvs
 #####################################################
-%define build_dvs 1
+#%%define build_dvs 1
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/debian.rules b/ultragrid-proprietary-drivers/debian.rules
--- a/ultragrid-proprietary-drivers/debian.rules	2017-12-15 17:14:35.122354194 +0100
+++ b/ultragrid-proprietary-drivers/debian.rules	2017-12-15 17:14:36.434378801 +0100
@@ -54,10 +54,10 @@ override_dh_auto_install:
 #####################################################
 # > dvs
 #####################################################
-	cp -r sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
-	ln -s sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk
-
-	rm -r $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk/linux-x86
+#	cp -r sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
+#	ln -s sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk
+#
+#	rm -r $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk/linux-x86
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
--- a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-12-15 17:14:35.122354194 +0100
+++ b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-12-15 17:14:36.310376476 +0100
@@ -72,8 +72,8 @@ Patch29:	AJA-kernel-backports-opensuse-4
 #####################################################
 # > dvs (sdk)
 #####################################################
-Patch30:	dvs-linux4.6-get-user-pages.patch
-Patch39:	dvs-kernel-backports-opensuse-423.patch
+#Patch30:	dvs-linux4.6-get-user-pages.patch
+#Patch39:	dvs-kernel-backports-opensuse-423.patch
 #####################################################
 # < dvs (sdk)
 #####################################################
@@ -130,10 +130,10 @@ VideoMasterHD		--deltacast
 #####################################################
 # > dvs (sdk)
 #####################################################
-%patch30 -p1
-%if 0%{?is_opensuse} >= 1 && 0%{?sle_version} >= 120200
-%patch39 -p1
-%endif
+#%%patch30 -p1
+#%%if 0%%{?is_opensuse} >= 1 && 0%%{?sle_version} >= 120200
+#%%patch39 -p1
+#%%endif
 #####################################################
 # < dvs (sdk)
 #####################################################
@@ -193,10 +193,10 @@ popd
 #####################################################
 # > dvs
 #####################################################
-cp -r sdk4.3.* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
-ln -s sdk4.3* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk
-
-rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk/linux-x86
+#cp -r sdk4.3.* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
+#ln -s sdk4.3* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk
+#
+#rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk/linux-x86
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers-1.4/debian.rules b/ultragrid-proprietary-drivers-1.4/debian.rules
--- a/ultragrid-proprietary-drivers-1.4/debian.rules	2017-12-15 17:14:35.126354268 +0100
+++ b/ultragrid-proprietary-drivers-1.4/debian.rules	2017-12-15 17:14:36.562381202 +0100
@@ -50,10 +50,10 @@ override_dh_auto_install:
 #####################################################
 # > dvs
 #####################################################
-	cp -r sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
-	ln -s sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk
-
-	rm -r $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk/linux-x86
+#	cp -r sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
+#	ln -s sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk
+#
+#	rm -r $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk/linux-x86
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
--- a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec	2017-12-15 17:14:35.126354268 +0100
+++ b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec	2017-12-15 17:14:36.458379252 +0100
@@ -64,7 +64,7 @@ Patch25:	AJA-linux4.6-get-user-pages.pat
 #####################################################
 # > dvs (sdk)
 #####################################################
-Patch30:	dvs-linux4.6-get-user-pages.patch
+#Patch30:	dvs-linux4.6-get-user-pages.patch
 #####################################################
 # < dvs (sdk)
 #####################################################
@@ -112,7 +112,7 @@ VideoMasterHD		--deltacast
 #####################################################
 # > dvs (sdk)
 #####################################################
-%patch30 -p1
+#%%patch30 -p1
 #####################################################
 # < dvs (sdk)
 #####################################################
@@ -168,10 +168,10 @@ popd
 #####################################################
 # > dvs
 #####################################################
-cp -r sdk4.3.* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
-ln -s sdk4.3* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk
-
-rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk/linux-x86
+#cp -r sdk4.3.* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
+#ln -s sdk4.3* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk
+#
+#rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk/linux-x86
 #####################################################
 # < dvs
 #####################################################
