diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules
+++ b/ultragrid/debian.rules
@@ -28,7 +28,7 @@ CARDCONF_DVS=--enable-dvs --with-dvs=/us
 #####################################################
 # > blackmagick
 #####################################################
-CARDCONF_DECKLINK= --enable-decklink
+#CARDCONF_DECKLINK= --enable-decklink
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl
+++ b/ultragrid/ultragrid.spec.tpl
@@ -116,7 +116,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > blackmagick
 #####################################################
-%define build_blackmagick 1
+#%%define build_blackmagick 1
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules
+++ b/ultragrid-nightly/debian.rules
@@ -28,7 +28,7 @@ CARDCONF_DVS=--enable-dvs --with-dvs=/us
 #####################################################
 # > blackmagick
 #####################################################
-CARDCONF_DECKLINK= --enable-decklink
+#CARDCONF_DECKLINK= --enable-decklink
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl
@@ -132,7 +132,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > blackmagick
 #####################################################
-%define build_blackmagick 1
+#%%define build_blackmagick 1
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
--- a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
+++ b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
@@ -18,7 +18,7 @@ BuildRequires:	%kernel_module_package_bu
 #####################################################
 # > blackmagick (DesktopVideo)
 #####################################################
-Requires:	desktopvideo
+#Requires:	desktopvideo
 #####################################################
 # < blackmagick (Desktopvideo)
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
--- a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
+++ b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
@@ -16,7 +16,7 @@ BuildRequires:	%kernel_module_package_bu
 #####################################################
 # > blackmagick (DesktopVideo)
 #####################################################
-Requires:	desktopvideo
+#Requires:	desktopvideo
 #####################################################
 # < blackmagick (Desktopvideo)
 #####################################################
