diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules
+++ b/ultragrid/debian.rules
@@ -51,7 +51,7 @@ CARDCONF_AJA= --enable-aja --with-aja=/u
 #####################################################
 # > cuda
 #####################################################
-CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
+#CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
 #####################################################
 # < cuda
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl
+++ b/ultragrid/ultragrid.spec.tpl
@@ -40,7 +40,7 @@ BuildRequires:	glib2-devel, libcurl-deve
 #####################################################
 # > cuda
 #####################################################
-%define cuda 1
+#%%define cuda 1
 #####################################################
 # < cuda
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules
+++ b/ultragrid-nightly/debian.rules
@@ -51,7 +51,7 @@ CARDCONF_AJA= --enable-aja --with-aja=/u
 #####################################################
 # > cuda
 #####################################################
-CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
+#CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
 #####################################################
 # < cuda
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl
@@ -40,7 +40,7 @@ BuildRequires:	glib2-devel, libcurl-deve
 #####################################################
 # > cuda
 #####################################################
-%define cuda 1
+#%%define cuda 1
 #####################################################
 # < cuda
 #####################################################
