diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules
+++ b/ultragrid/debian.rules
@@ -35,7 +35,7 @@ CARDCONF_DECKLINK= --enable-decklink
 #####################################################
 # > deltacast
 #####################################################
-CARDCONF_DELTACAST= --enable-deltacast --with-deltacast=/usr/src/ultragrid-externals/deltacast_sdk
+#CARDCONF_DELTACAST= --enable-deltacast --with-deltacast=/usr/src/ultragrid-externals/deltacast_sdk
 #####################################################
 # < deltacast
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl
+++ b/ultragrid/ultragrid.spec.tpl
@@ -123,7 +123,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > deltacast
 #####################################################
-%define build_deltacast 1
+#%%define build_deltacast 1
 #####################################################
 # < deltacast
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules
+++ b/ultragrid-nightly/debian.rules
@@ -35,7 +35,7 @@ CARDCONF_DECKLINK= --enable-decklink
 #####################################################
 # > deltacast
 #####################################################
-CARDCONF_DELTACAST= --enable-deltacast --with-deltacast=/usr/src/ultragrid-externals/deltacast_sdk
+#CARDCONF_DELTACAST= --enable-deltacast --with-deltacast=/usr/src/ultragrid-externals/deltacast_sdk
 #####################################################
 # < deltacast
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl
@@ -139,7 +139,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > deltacast
 #####################################################
-%define build_deltacast 1
+#%%define build_deltacast 1
 #####################################################
 # < deltacast
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/debian.rules b/ultragrid-proprietary-drivers/debian.rules
--- a/ultragrid-proprietary-drivers/debian.rules
+++ b/ultragrid-proprietary-drivers/debian.rules
@@ -72,14 +72,14 @@ override_dh_auto_install:
 #####################################################
 # > deltacast
 #####################################################
-	cp -r VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
-	ln -s VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk
-
-	#deltacast
-	mkdir -p $(DEB_BUILD_ROOT)/usr/lib
-	cd VideoMasterHD_*/Library/ ; for i in $$(ls -d */x64/); do pushd $$i ; echo $$i ; make install libdir=$(DEB_BUILD_ROOT)/usr/lib ; popd ; done
-
-	find $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r '{}' \;
+#	cp -r VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
+#	ln -s VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk
+#
+#	#deltacast
+#	mkdir -p $(DEB_BUILD_ROOT)/usr/lib
+#	cd VideoMasterHD_*/Library/ ; for i in $$(ls -d */x64/); do pushd $$i ; echo $$i ; make install libdir=$(DEB_BUILD_ROOT)/usr/lib ; popd ; done
+#
+#	find $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r '{}' \;
 #####################################################
 # < deltacast
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
--- a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
+++ b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
@@ -36,13 +36,13 @@ Source1:	ultragrid-proprietary-drivers-r
 #####################################################
 # > deltacast (videoMasterHD)
 #####################################################
-Patch0:		videoMasterHD-destdir.patch
-#Patch1:		videoMasterHD-linux4.6-get-user-pages.patch
-#Patch9:		videoMasterHD-kernel-backports-opensuse-423.patch
-# somehow, provides detection seems to be broken on Fedora
-%if 0%{?fedora}
-Provides:	libFlxComm64.so()(64bit)
-%endif
+#Patch0:		videoMasterHD-destdir.patch
+##Patch1:		videoMasterHD-linux4.6-get-user-pages.patch
+##Patch9:		videoMasterHD-kernel-backports-opensuse-423.patch
+## somehow, provides detection seems to be broken on Fedora
+#%%if 0%%{?fedora}
+#Provides:	libFlxComm64.so()(64bit)
+#%%endif
 #####################################################
 # < deltacast (videoMasterHD)
 #####################################################
@@ -92,11 +92,11 @@ VideoMasterHD		--deltacast
 #####################################################
 # > deltacast
 #####################################################
-%patch0 -p1
-#%patch1 -p1
-#%if 0%{?is_opensuse} >= 1 && 0%{?sle_version} == 120300
-#%patch9 -p1
-#%endif
+#%%patch0 -p1
+##%patch1 -p1
+##%if 0%{?is_opensuse} >= 1 && 0%{?sle_version} == 120300
+##%patch9 -p1
+##%endif
 #####################################################
 # < deltacast
 #####################################################
@@ -213,23 +213,23 @@ ln -s ntv2sdklinux_* $RPM_BUILD_ROOT/usr
 #####################################################
 # > deltacast
 #####################################################
-cp -r VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
-ln -s VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/deltacast_sdk
-
-mkdir -p $RPM_BUILD_ROOT%{_libdir}
-pushd VideoMasterHD_*/Library/
-for i in $(ls -d */x64/)
-do
-    pushd $i
-    env libdir=%{_libdir} %makeinstall
-    popd
-done
-popd
-
-# fix bad arch binaries
-find ${RPM_BUILD_ROOT}/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r {} \;
-find ${RPM_BUILD_ROOT}/ -executable -type f -exec file {} \; | grep -i 'elf 32' | cut -d':' -f1 | while read -r -d $'\n' filename ; do rm "$filename" ; done
-find ${RPM_BUILD_ROOT}/ -iregex '.*\.so\(\.[0-9]+\)*$' -type f -exec file {} \; | grep -i 'elf 32' | cut -d':' -f1 | while read -r -d $'\n' filename ; do rm "$filename" ; done
+#cp -r VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
+#ln -s VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/deltacast_sdk
+#
+#mkdir -p $RPM_BUILD_ROOT%{_libdir}
+#pushd VideoMasterHD_*/Library/
+#for i in $(ls -d */x64/)
+#do
+#    pushd $i
+#    env libdir=%{_libdir} %makeinstall
+#    popd
+#done
+#popd
+#
+## fix bad arch binaries
+#find ${RPM_BUILD_ROOT}/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r {} \;
+#find ${RPM_BUILD_ROOT}/ -executable -type f -exec file {} \; | grep -i 'elf 32' | cut -d':' -f1 | while read -r -d $'\n' filename ; do rm "$filename" ; done
+#find ${RPM_BUILD_ROOT}/ -iregex '.*\.so\(\.[0-9]+\)*$' -type f -exec file {} \; | grep -i 'elf 32' | cut -d':' -f1 | while read -r -d $'\n' filename ; do rm "$filename" ; done
 #####################################################
 # < deltacast
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers-1.4/debian.rules b/ultragrid-proprietary-drivers-1.4/debian.rules
--- a/ultragrid-proprietary-drivers-1.4/debian.rules
+++ b/ultragrid-proprietary-drivers-1.4/debian.rules
@@ -68,14 +68,14 @@ override_dh_auto_install:
 #####################################################
 # > deltacast
 #####################################################
-	cp -r VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
-	ln -s VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk
-
-	#deltacast
-	mkdir -p $(DEB_BUILD_ROOT)/usr/lib
-	cd VideoMasterHD_*/Library/ ; for i in $$(ls -d */x64/); do pushd $$i ; echo $$i ; make install libdir=$(DEB_BUILD_ROOT)/usr/lib ; popd ; done
-
-	find $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r '{}' \;
+#	cp -r VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
+#	ln -s VideoMasterHD_* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk
+#
+#	#deltacast
+#	mkdir -p $(DEB_BUILD_ROOT)/usr/lib
+#	cd VideoMasterHD_*/Library/ ; for i in $$(ls -d */x64/); do pushd $$i ; echo $$i ; make install libdir=$(DEB_BUILD_ROOT)/usr/lib ; popd ; done
+#
+#	find $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r '{}' \;
 #####################################################
 # < deltacast
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
--- a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
+++ b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
@@ -34,8 +34,8 @@ Source1:	ultragrid-proprietary-drivers-1
 #####################################################
 # > deltacast (videoMasterHD)
 #####################################################
-Patch0:		videoMasterHD-destdir.patch
-Patch1:		videoMasterHD-linux4.6-get-user-pages.patch
+#Patch0:		videoMasterHD-destdir.patch
+#Patch1:		videoMasterHD-linux4.6-get-user-pages.patch
 #####################################################
 # < deltacast (videoMasterHD)
 #####################################################
@@ -85,8 +85,8 @@ VideoMasterHD		--deltacast
 #####################################################
 # > deltacast
 #####################################################
-%patch0 -p1
-%patch1 -p1
+#%%patch0 -p1
+#%%patch1 -p1
 #####################################################
 # < deltacast
 #####################################################
@@ -198,21 +198,21 @@ ln -s ntv2sdklinux_* $RPM_BUILD_ROOT/usr
 #####################################################
 # > deltacast
 #####################################################
-cp -r VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
-ln -s VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/deltacast_sdk
-
-mkdir -p $RPM_BUILD_ROOT%{_libdir}
-pushd VideoMasterHD_*/Library/
-for i in $(ls -d */x64/)
-do
-    pushd $i
-    env libdir=%{_libdir} %makeinstall
-    popd
-done
-popd
-
-# fix bad arch binaries
-find ${RPM_BUILD_ROOT}/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r {} \;
+#cp -r VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
+#ln -s VideoMasterHD* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/deltacast_sdk
+#
+#mkdir -p $RPM_BUILD_ROOT%{_libdir}
+#pushd VideoMasterHD_*/Library/
+#for i in $(ls -d */x64/)
+#do
+#    pushd $i
+#    env libdir=%{_libdir} %makeinstall
+#    popd
+#done
+#popd
+#
+## fix bad arch binaries
+#find ${RPM_BUILD_ROOT}/usr/src/ultragrid-externals/deltacast_sdk/Library/ -maxdepth 2 -name x86 -exec rm -r {} \;
 #####################################################
 # < deltacast
 #####################################################
