diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules	2017-04-07 13:20:59.342214183 +0200
+++ b/ultragrid/debian.rules	2017-04-07 13:21:00.422234200 +0200
@@ -14,7 +14,7 @@ CUDA=--disable-cuda
 #####################################################
 # > bluefish
 #####################################################
-CARDCONF_BLUEFISH=--enable-bluefish444 --enable-blue-audio --with-bluefish444=/usr/src/ultragrid-externals/bluefish_sdk
+#CARDCONF_BLUEFISH=--enable-bluefish444 --enable-blue-audio --with-bluefish444=/usr/src/ultragrid-externals/bluefish_sdk
 #####################################################
 # < bluefish
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl	2017-04-07 13:20:59.342214183 +0200
+++ b/ultragrid/ultragrid.spec.tpl	2017-04-07 13:21:00.386233534 +0200
@@ -101,7 +101,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > bluefish
 #####################################################
-%define build_bluefish 1
+#%%define build_bluefish 1
 #####################################################
 # < bluefish
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules	2017-04-07 13:20:59.338214108 +0200
+++ b/ultragrid-nightly/debian.rules	2017-04-07 13:21:00.366233163 +0200
@@ -14,7 +14,7 @@ CUDA=--disable-cuda
 #####################################################
 # > bluefish
 #####################################################
-CARDCONF_BLUEFISH=--enable-bluefish444 --enable-blue-audio --with-bluefish444=/usr/src/ultragrid-externals/bluefish_sdk
+#CARDCONF_BLUEFISH=--enable-bluefish444 --enable-blue-audio --with-bluefish444=/usr/src/ultragrid-externals/bluefish_sdk
 #####################################################
 # < bluefish
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-04-07 13:20:59.338214108 +0200
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-04-07 13:21:00.330232496 +0200
@@ -101,7 +101,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > bluefish
 #####################################################
-%define build_bluefish 1
+#%%define build_bluefish 1
 #####################################################
 # < bluefish
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/debian.rules b/ultragrid-proprietary-drivers/debian.rules
--- a/ultragrid-proprietary-drivers/debian.rules	2017-04-07 13:20:59.346214257 +0200
+++ b/ultragrid-proprietary-drivers/debian.rules	2017-04-07 13:21:00.538236350 +0200
@@ -25,8 +25,8 @@ override_dh_auto_build:
 #####################################################
 # > bluefish
 #####################################################
-	cd EpochLinuxDriver_V5*/drivers/orac ; env libdir=/usr/lib make
-	cd EpochLinuxDriver_V5*/apis/BlueVelvet ; env libdir=/usr/lib make
+#	cd EpochLinuxDriver_V5*/drivers/orac ; env libdir=/usr/lib make
+#	cd EpochLinuxDriver_V5*/apis/BlueVelvet ; env libdir=/usr/lib make
 #####################################################
 # < bluefish
 #####################################################
@@ -39,11 +39,11 @@ override_dh_auto_install:
 #####################################################
 # > bluefish
 #####################################################
-	cp -r EpochLinuxDriver_V5* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
-	ln -s EpochLinuxDriver_V5* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/bluefish_sdk
-
-	cd EpochLinuxDriver_V5*/drivers/orac ; env libdir=/usr/lib make install DESTDIR=$(DEB_BUILD_ROOT)
-	cd EpochLinuxDriver_V5*/apis/BlueVelvet ; env libdir=/usr/lib make install DESTDIR=$(DEB_BUILD_ROOT)
+#	cp -r EpochLinuxDriver_V5* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
+#	ln -s EpochLinuxDriver_V5* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/bluefish_sdk
+#
+#	cd EpochLinuxDriver_V5*/drivers/orac ; env libdir=/usr/lib make install DESTDIR=$(DEB_BUILD_ROOT)
+#	cd EpochLinuxDriver_V5*/apis/BlueVelvet ; env libdir=/usr/lib make install DESTDIR=$(DEB_BUILD_ROOT)
 #####################################################
 # < bluefish
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
--- a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-04-07 13:20:59.346214257 +0200
+++ b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-04-07 13:21:00.446234646 +0200
@@ -39,10 +39,10 @@ Patch1:		videoMasterHD-linux4.6-get-user
 #####################################################
 # > bluefish (EpochLinuxDriver)
 #####################################################
-Patch10:	bluefish-uname.patch
-Patch11:	bluefish-g++.patch
-Patch12:	bluefish-destdir.patch
-Patch13:	bluefish-linux4.6-get-user-pages.patch
+#Patch10:	bluefish-uname.patch
+#Patch11:	bluefish-g++.patch
+#Patch12:	bluefish-destdir.patch
+#Patch13:	bluefish-linux4.6-get-user-pages.patch
 #####################################################
 # < bluefish (EpochLinuxDriver)
 #####################################################
@@ -87,10 +87,10 @@ VideoMasterHD		--deltacast
 #####################################################
 # > bluefish
 #####################################################
-%patch10 -p1
-%patch11 -p1
-%patch12 -p1
-%patch13 -p1
+#%%patch10 -p1
+#%%patch11 -p1
+#%%patch12 -p1
+#%%patch13 -p1
 #####################################################
 # < bluefish
 #####################################################
@@ -131,12 +131,12 @@ popd
 #####################################################
 # > bluefish
 #####################################################
-pushd EpochLinuxDriver_V5*/drivers/orac
-env libdir=%{_libdir} make
-popd
-pushd EpochLinuxDriver_V5*/apis/BlueVelvet
-env libdir=%{_libdir} make
-popd
+#pushd EpochLinuxDriver_V5*/drivers/orac
+#env libdir=%{_libdir} make
+#popd
+#pushd EpochLinuxDriver_V5*/apis/BlueVelvet
+#env libdir=%{_libdir} make
+#popd
 #####################################################
 # < bluefish
 #####################################################
@@ -150,15 +150,15 @@ mkdir -p $RPM_BUILD_ROOT/usr/src/ultragr
 #####################################################
 # > bluefish
 #####################################################
-cp -r EpochLinuxDriver_V5* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
-ln -s EpochLinuxDriver_V5* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/bluefish_sdk
-
-pushd EpochLinuxDriver_V5*/drivers/orac
-env libdir=%{_libdir} make install DESTDIR=$RPM_BUILD_ROOT
-popd
-pushd EpochLinuxDriver_V5*/apis/BlueVelvet
-env libdir=%{_libdir} make install DESTDIR=$RPM_BUILD_ROOT
-popd
+#cp -r EpochLinuxDriver_V5* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
+#ln -s EpochLinuxDriver_V5* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/bluefish_sdk
+#
+#pushd EpochLinuxDriver_V5*/drivers/orac
+#env libdir=%{_libdir} make install DESTDIR=$RPM_BUILD_ROOT
+#popd
+#pushd EpochLinuxDriver_V5*/apis/BlueVelvet
+#env libdir=%{_libdir} make install DESTDIR=$RPM_BUILD_ROOT
+#popd
 #####################################################
 # < bluefish
 #####################################################
