diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules	2017-05-19 15:14:28.700837150 +0200
+++ b/ultragrid/debian.rules	2017-05-19 15:14:30.616843567 +0200
@@ -49,7 +49,7 @@ CARDCONF_AJA= --enable-aja --with-aja=/u
 #####################################################
 # > cuda
 #####################################################
-CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
+#CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
 #####################################################
 # < cuda
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl	2017-05-19 15:14:28.700837150 +0200
+++ b/ultragrid/ultragrid.spec.tpl	2017-05-19 15:14:30.568843406 +0200
@@ -35,7 +35,7 @@ BuildRequires:	glib2-devel, libcurl-deve
 #####################################################
 # > cuda
 #####################################################
-%define cuda 1
+#%%define cuda 1
 #####################################################
 # < cuda
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules	2017-05-19 15:14:28.688837109 +0200
+++ b/ultragrid-nightly/debian.rules	2017-05-19 15:14:30.544843326 +0200
@@ -49,7 +49,7 @@ CARDCONF_AJA= --enable-aja --with-aja=/u
 #####################################################
 # > cuda
 #####################################################
-CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
+#CUDA=--with-cuda=/usr/local/cuda-8.0  --enable-jpeg
 #####################################################
 # < cuda
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-05-19 15:14:28.688837109 +0200
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-05-19 15:14:30.496843166 +0200
@@ -35,7 +35,7 @@ BuildRequires:	glib2-devel, libcurl-deve
 #####################################################
 # > cuda
 #####################################################
-%define cuda 1
+#%%define cuda 1
 #####################################################
 # < cuda
 #####################################################
