diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules	2017-04-07 13:20:59.342214183 +0200
+++ b/ultragrid/debian.rules	2017-04-07 13:20:59.946225377 +0200
@@ -21,7 +21,7 @@ CARDCONF_BLUEFISH=--enable-bluefish444 -
 #####################################################
 # > dvs
 #####################################################
-CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
+#CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl	2017-04-07 13:20:59.342214183 +0200
+++ b/ultragrid/ultragrid.spec.tpl	2017-04-07 13:20:59.906224637 +0200
@@ -108,7 +108,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > dvs
 #####################################################
-%define build_dvs 1
+#%%define build_dvs 1
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules	2017-04-07 13:20:59.338214108 +0200
+++ b/ultragrid-nightly/debian.rules	2017-04-07 13:20:59.886224266 +0200
@@ -21,7 +21,7 @@ CARDCONF_BLUEFISH=--enable-bluefish444 -
 #####################################################
 # > dvs
 #####################################################
-CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
+#CARDCONF_DVS=--enable-dvs --with-dvs=/usr/src/ultragrid-externals/dvs_sdk
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-04-07 13:20:59.338214108 +0200
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl	2017-04-07 13:20:59.850223598 +0200
@@ -108,7 +108,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > dvs
 #####################################################
-%define build_dvs 1
+#%%define build_dvs 1
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/debian.rules b/ultragrid-proprietary-drivers/debian.rules
--- a/ultragrid-proprietary-drivers/debian.rules	2017-04-07 13:20:59.346214257 +0200
+++ b/ultragrid-proprietary-drivers/debian.rules	2017-04-07 13:21:00.062227527 +0200
@@ -50,10 +50,10 @@ override_dh_auto_install:
 #####################################################
 # > dvs
 #####################################################
-	cp -r sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
-	ln -s sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk
-
-	rm -r $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk/linux-x86
+#	cp -r sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
+#	ln -s sdk4.3* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk
+#
+#	rm -r $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/dvs_sdk/linux-x86
 #####################################################
 # < dvs
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
--- a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-04-07 13:20:59.346214257 +0200
+++ b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-04-07 13:20:59.966225748 +0200
@@ -61,7 +61,7 @@ Patch25:	AJA-linux4.6-get-user-pages.pat
 #####################################################
 # > dvs (sdk)
 #####################################################
-Patch30:	dvs-linux4.6-get-user-pages.patch
+#Patch30:	dvs-linux4.6-get-user-pages.patch
 #####################################################
 # < dvs (sdk)
 #####################################################
@@ -109,7 +109,7 @@ VideoMasterHD		--deltacast
 #####################################################
 # > dvs (sdk)
 #####################################################
-%patch30 -p1
+#%%patch30 -p1
 #####################################################
 # < dvs (sdk)
 #####################################################
@@ -165,10 +165,10 @@ popd
 #####################################################
 # > dvs
 #####################################################
-cp -r sdk4.3.* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
-ln -s sdk4.3* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk
-
-rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk/linux-x86
+#cp -r sdk4.3.* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
+#ln -s sdk4.3* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk
+#
+#rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-externals/dvs_sdk/linux-x86
 #####################################################
 # < dvs
 #####################################################
