diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules
+++ b/ultragrid/debian.rules
@@ -42,7 +42,7 @@ CARDCONF_DELTACAST= --enable-deltacast -
 #####################################################
 # > aja
 #####################################################
-CARDCONF_AJA= --enable-aja --with-aja=/usr/src/ultragrid-externals/aja_sdk
+#CARDCONF_AJA= --enable-aja --with-aja=/usr/src/ultragrid-externals/aja_sdk
 #####################################################
 # < aja
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec.tpl b/ultragrid/ultragrid.spec.tpl
--- a/ultragrid/ultragrid.spec.tpl
+++ b/ultragrid/ultragrid.spec.tpl
@@ -130,7 +130,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > aja
 #####################################################
-%define build_aja 1
+#%%define build_aja 1
 #####################################################
 # < aja
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules
+++ b/ultragrid-nightly/debian.rules
@@ -44,7 +44,7 @@ CARDCONF_DELTACAST= --enable-deltacast -
 #####################################################
 # > aja
 #####################################################
-CARDCONF_AJA= --enable-aja --with-aja=/usr/src/ultragrid-externals/aja_sdk
+#CARDCONF_AJA= --enable-aja --with-aja=/usr/src/ultragrid-externals/aja_sdk
 #####################################################
 # < aja
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec.tpl b/ultragrid-nightly/ultragrid-nightly.spec.tpl
--- a/ultragrid-nightly/ultragrid-nightly.spec.tpl
+++ b/ultragrid-nightly/ultragrid-nightly.spec.tpl
@@ -153,7 +153,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > aja
 #####################################################
-%define build_aja 1
+#%%define build_aja 1
 #####################################################
 # < aja
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/debian.rules b/ultragrid-proprietary-drivers/debian.rules
--- a/ultragrid-proprietary-drivers/debian.rules
+++ b/ultragrid-proprietary-drivers/debian.rules
@@ -21,7 +21,7 @@ override_dh_auto_build:
 #####################################################
 # > aja
 #####################################################
-	cd ntv2sdk* ; make QTDIR=/usr/lib/qt5 AJA_NO_FLTK=1
+#	cd ntv2sdk* ; make QTDIR=/usr/lib/qt5 AJA_NO_FLTK=1
 #####################################################
 # < aja
 #####################################################
@@ -65,8 +65,8 @@ override_dh_auto_install:
 #####################################################
 # > aja
 #####################################################
-	ln -s ntv2sdklinux* aja_sdk
-	tar -c aja_sdk ntv2sdklinux* -f - | tar -C $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/ -xf -
+#	ln -s ntv2sdklinux* aja_sdk
+#	tar -c aja_sdk ntv2sdklinux* -f - | tar -C $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/ -xf -
 #####################################################
 # < aja
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
--- a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
+++ b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
@@ -60,13 +60,13 @@ Patch19:	bluefish-kernel-backports-opens
 #####################################################
 # > aja (ntv2sdklinux)
 #####################################################
-Patch20:	AJA-linuxdriver-uname.patch
-Patch21:	AJA-nodemo.patch
-Patch22:	AJA-qmake.patch
-Patch23:	AJA-qt5.patch
-Patch24:	AJA-gcc-explicit-constructors.patch
-Patch25:	AJA-linux4.16-flush-write-buffers.patch
-Patch29:	AJA-kernel-backports-opensuse-423.patch
+#Patch20:	AJA-linuxdriver-uname.patch
+#Patch21:	AJA-nodemo.patch
+#Patch22:	AJA-qmake.patch
+#Patch23:	AJA-qt5.patch
+#Patch24:	AJA-gcc-explicit-constructors.patch
+#Patch25:	AJA-linux4.16-flush-write-buffers.patch
+#Patch29:	AJA-kernel-backports-opensuse-423.patch
 #####################################################
 # < aja (ntv2sdklinux)
 #####################################################
@@ -121,16 +121,16 @@ VideoMasterHD		--deltacast
 #####################################################
 # > aja
 #####################################################
-%patch20 -p1
-%patch21 -p1
-%patch22 -p1
-%patch23 -p1
-%patch24 -p1
-%if 0%{?is_opensuse} >= 1 && 0%{?sle_version} == 120300
-%patch29 -p1
-%endif
-#on intention
-%patch25 -p1
+#%%patch20 -p1
+#%%patch21 -p1
+#%%patch22 -p1
+#%%patch23 -p1
+#%%patch24 -p1
+#%%if 0%%{?is_opensuse} >= 1 && 0%%{?sle_version} == 120300
+#%%patch29 -p1
+#%%endif
+##on intention
+#%%patch25 -p1
 #####################################################
 # < aja
 #####################################################
@@ -154,9 +154,9 @@ find . -type f -name "*~" -exec rm {} \;
 #####################################################
 # > aja
 #####################################################
-pushd ntv2sdk*
-env libdir=%{_libdir} make QTDIR=/usr/lib/qt5 AJA_NO_FLTK=1
-popd
+#pushd ntv2sdk*
+#env libdir=%{_libdir} make QTDIR=/usr/lib/qt5 AJA_NO_FLTK=1
+#popd
 #####################################################
 # < aja
 #####################################################
@@ -210,8 +210,8 @@ rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-
 #####################################################
 # > aja
 #####################################################
-ln -s ntv2sdklinux_* aja_sdk
-tar -c aja_sdk ntv2sdklinux_* -f - | tar -C $RPM_BUILD_ROOT/usr/src/ultragrid-externals/ -xf -
+#ln -s ntv2sdklinux_* aja_sdk
+#tar -c aja_sdk ntv2sdklinux_* -f - | tar -C $RPM_BUILD_ROOT/usr/src/ultragrid-externals/ -xf -
 #####################################################
 # < aja
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers-1.4/debian.rules b/ultragrid-proprietary-drivers-1.4/debian.rules
--- a/ultragrid-proprietary-drivers-1.4/debian.rules
+++ b/ultragrid-proprietary-drivers-1.4/debian.rules
@@ -18,7 +18,7 @@ override_dh_auto_build:
 #####################################################
 # > aja
 #####################################################
-	cd ntv2sdklinux*/ntv2projects ; make QTDIR=/usr/lib/qt5
+#	cd ntv2sdklinux*/ntv2projects ; make QTDIR=/usr/lib/qt5
 #####################################################
 # < aja
 #####################################################
@@ -60,8 +60,8 @@ override_dh_auto_install:
 #####################################################
 # > aja
 #####################################################
-	cp -r ntv2sdklinux* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
-	ln -s ntv2sdklinux* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/aja_sdk
+#	cp -r ntv2sdklinux* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/
+#	ln -s ntv2sdklinux* $(DEB_BUILD_ROOT)/usr/src/ultragrid-externals/aja_sdk
 #####################################################
 # < aja
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
--- a/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
+++ b/ultragrid-proprietary-drivers-1.4/ultragrid-proprietary-drivers-1.4.spec
@@ -53,13 +53,13 @@ Patch19:	bluefish-kernel-backports-opens
 #####################################################
 # > aja (ntv2sdklinux)
 #####################################################
-Patch20:	AJA-linuxdriver-uname.patch
-Patch21:	AJA-nodemo.patch
-Patch22:	AJA-qmake.patch
-Patch23:	AJA-qt5.patch
-Patch24:	AJA-gcc-explicit-constructors.patch
-Patch25:	AJA-linux4.6-get-user-pages.patch
-Patch29:	AJA-kernel-backports-opensuse-423.patch
+#Patch20:	AJA-linuxdriver-uname.patch
+#Patch21:	AJA-nodemo.patch
+#Patch22:	AJA-qmake.patch
+#Patch23:	AJA-qt5.patch
+#Patch24:	AJA-gcc-explicit-constructors.patch
+#Patch25:	AJA-linux4.6-get-user-pages.patch
+#Patch29:	AJA-kernel-backports-opensuse-423.patch
 #####################################################
 # < aja (ntv2sdklinux)
 #####################################################
@@ -106,15 +106,15 @@ VideoMasterHD		--deltacast
 #####################################################
 # > aja
 #####################################################
-%patch20 -p1
-%patch21 -p1
-%patch22 -p1
-%patch23 -p1
-%patch24 -p1
-%patch25 -p1
-%if 0%{?is_opensuse} >= 1 && 0%{?sle_version} == 120300
-%patch29 -p1
-%endif
+#%%patch20 -p1
+#%%patch21 -p1
+#%%patch22 -p1
+#%%patch23 -p1
+#%%patch24 -p1
+#%%patch25 -p1
+#%%if 0%%{?is_opensuse} >= 1 && 0%%{?sle_version} == 120300
+#%%patch29 -p1
+#%%endif
 #####################################################
 # < aja
 #####################################################
@@ -135,10 +135,10 @@ VideoMasterHD		--deltacast
 #####################################################
 # > aja
 #####################################################
-ls
-pushd ntv2sdklinux_*/ntv2projects
-env libdir=%{_libdir} make QTDIR=/usr/lib/qt5
-popd
+#ls
+#pushd ntv2sdklinux_*/ntv2projects
+#env libdir=%{_libdir} make QTDIR=/usr/lib/qt5
+#popd
 #####################################################
 # < aja
 #####################################################
@@ -190,8 +190,8 @@ rm -r $RPM_BUILD_ROOT/usr/src/ultragrid-
 #####################################################
 # > aja
 #####################################################
-cp -r ntv2sdklinux_* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
-ln -s ntv2sdklinux_* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/aja_sdk
+#cp -r ntv2sdklinux_* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/
+#ln -s ntv2sdklinux_* $RPM_BUILD_ROOT/usr/src/ultragrid-externals/aja_sdk
 #####################################################
 # < aja
 #####################################################
