diff -rupN a/ultragrid/debian.rules b/ultragrid/debian.rules
--- a/ultragrid/debian.rules	2017-04-06 12:53:03.317244490 +0200
+++ b/ultragrid/debian.rules	2017-04-06 12:53:03.641249875 +0200
@@ -21,7 +21,7 @@ CARDCONF+= --enable-dvs --with-dvs=/usr/
 #####################################################
 # > blackmagick
 #####################################################
-CARDCONF+= --enable-decklink
+#CARDCONF+= --enable-decklink
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid/ultragrid.spec b/ultragrid/ultragrid.spec
--- a/ultragrid/ultragrid.spec	2017-04-06 12:53:03.317244490 +0200
+++ b/ultragrid/ultragrid.spec	2017-04-06 12:53:03.677250474 +0200
@@ -102,7 +102,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > blackmagick
 #####################################################
-%define build_blackmagick 1
+#%define build_blackmagick 1
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid-nightly/debian.rules b/ultragrid-nightly/debian.rules
--- a/ultragrid-nightly/debian.rules	2017-04-06 12:53:03.313244424 +0200
+++ b/ultragrid-nightly/debian.rules	2017-04-06 12:53:03.581248878 +0200
@@ -21,7 +21,7 @@ CARDCONF+= --enable-dvs --with-dvs=/usr/
 #####################################################
 # > blackmagick
 #####################################################
-CARDCONF+= --enable-decklink
+#CARDCONF+= --enable-decklink
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid-nightly/ultragrid-nightly.spec b/ultragrid-nightly/ultragrid-nightly.spec
--- a/ultragrid-nightly/ultragrid-nightly.spec	2017-04-06 12:53:03.313244424 +0200
+++ b/ultragrid-nightly/ultragrid-nightly.spec	2017-04-06 12:53:03.617249476 +0200
@@ -101,7 +101,7 @@ UltraGrid developed by Colin Perkins, La
 #####################################################
 # > blackmagick
 #####################################################
-%define build_blackmagick 1
+#%define build_blackmagick 1
 #####################################################
 # < blackmagick
 #####################################################
diff -rupN a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec
--- a/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-04-06 12:53:03.321244557 +0200
+++ b/ultragrid-proprietary-drivers/ultragrid-proprietary-drivers.spec	2017-04-06 12:53:03.737251470 +0200
@@ -13,7 +13,7 @@ BuildRequires:	%kernel_module_package_bu
 #####################################################
 # > blackmagick (DesktopVideo)
 #####################################################
-Requires:	desktopvideo
+#Requires:	desktopvideo
 #####################################################
 # < blackmagick (Desktopvideo)
 #####################################################
