--- a/configure.ac	2015-03-31 13:32:29.000000000 +0200
+++ b/configure.ac	2015-04-02 10:39:42.175596554 +0200
@@ -1986,7 +1986,7 @@ AC_ARG_WITH(cuda,
             ])
 
 AC_PATH_PROG(NVCC, nvcc, [], [$PATH$PATH_SEPARATOR$CUDA_PATH/bin]dnl
-        [$PATH_SEPARATOR/opt/cuda/bin$PATH_SEPARATOR/usr/local/cuda/bin])
+        [$PATH_SEPARATOR/opt/cuda/bin$PATH_SEPARATOR/usr/local/cuda/bin$PATH_SEPARATOR/usr/local/cuda-7.0/bin])
 
 if test -n "$NVCC" -a $cuda_req != no
 then
