diff -rupN a/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driverdma.c b/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driverdma.c
--- a/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driverdma.c	2015-03-25 13:34:04.000000000 +0100
+++ b/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driverdma.c	2016-04-27 21:50:41.034105019 +0200
@@ -25,7 +25,7 @@
 #include <asm/semaphore.h>
 #endif
 
-#include <asm/scatterlist.h>
+#include <linux/scatterlist.h>
 #include <linux/pci.h>
 #include <linux/bootmem.h>
 #include <linux/sched.h>
diff -rupN a/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driver.h b/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driver.h
--- a/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driver.h	2015-03-25 13:34:04.000000000 +0100
+++ b/ntv2linux-12.3.7.34/ntv2projects/linuxdriver/ntv2driver.h	2016-04-27 16:07:24.406906932 +0200
@@ -17,7 +17,7 @@
 #ifndef NTV2_DRIVER_HEADER
 #define NTV2_DRIVER_HEADER
 
-#include <asm/scatterlist.h>
+#include <linux/scatterlist.h>
 
 // Some kernel version sensitive macro-rama
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,22)
