diff -rupN a/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/watchermain.cpp b/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/watchermain.cpp
--- a/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/watchermain.cpp	2016-10-28 10:24:00.496000000 +0200
+++ b/ntv2sdklinux_12.4.2.1/ntv2projects/commonapps/ntv2watcher/watchermain.cpp	2016-11-10 11:48:54.187356863 +0100
@@ -48,8 +48,8 @@ NTV2Watcher::NTV2Watcher(QWidget *parent
     timer = new QTimer(this);
     connect(timer, &QTimer::timeout, this, &NTV2Watcher::slot_pollDevice);
 
-    QStyleHints * qsh = qApp->styleHints();
-    qsh->setStartDragTime(100);
+    //QStyleHints * qsh = qApp->styleHints();
+    qApp->setStartDragTime(100);
 
     connect (widgetList, &WidgetList::addRegister,  this, &NTV2Watcher::slot_addRegister,  Qt::QueuedConnection);
     connect (widgetList, &WidgetList::detachWidget, this, &NTV2Watcher::slot_detachWidget, Qt::QueuedConnection);
