diff -rupN a/sdk4.3.5.21/development/src_lucy/driver/linux.c b/sdk4.3.5.21/development/src_lucy/driver/linux.c
--- a/sdk4.3.5.21/development/src_lucy/driver/linux.c	2017-02-11 13:39:17.582479069 +0100
+++ b/sdk4.3.5.21/development/src_lucy/driver/linux.c	2017-02-11 15:31:36.411730454 +0100
@@ -790,7 +790,11 @@ INTERNAL donttag ps_alenlist * hal_LockM
 
   /*** make all pages present */
   down_read(&current->mm->mmap_sem);
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
   result = get_user_pages(current, current->mm, (unsigned long) addr, npages, bcard2host, 0 /* force */, pages, NULL);
+#else
+  result = get_user_pages((unsigned long) addr, npages, bcard2host, 0 /* force */, pages, NULL);
+#endif
   up_read(&current->mm->mmap_sem);
   if (result != npages) {
     DPF_WARN("hal_LockMemory: get_user_pages failed (%d)\n", result);
@@ -907,7 +911,11 @@ INTERNAL donttag void hal_UnlockMemory(v
         if (alenlist->flags & PS_ALENLIST_READ) {
           SetPageDirty(pages[i]);
         }
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
         page_cache_release(pages[i]);
+#else
+        put_page(pages[i]);
+#endif
       }
     }
 
diff -rupN a/sdk4.3.5.21/development/src_v4r3/driver/linux.c b/sdk4.3.5.21/development/src_v4r3/driver/linux.c
--- a/sdk4.3.5.21/development/src_v4r3/driver/linux.c	2017-02-11 13:39:17.582479069 +0100
+++ b/sdk4.3.5.21/development/src_v4r3/driver/linux.c	2017-02-11 15:31:36.415730649 +0100
@@ -806,7 +806,11 @@ INTERNAL donttag ps_alenlist * hal_LockM
 
   /*** make all pages present */
   down_read(&current->mm->mmap_sem);
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
   result = get_user_pages(current, current->mm, (unsigned long) addr, npages, bcard2host, 0 /* force */, pages, NULL);
+#else
+  result = get_user_pages((unsigned long) addr, npages, bcard2host, 0 /* force */, pages, NULL);
+#endif
   up_read(&current->mm->mmap_sem);
   if (result != npages) {
     DPF_WARN("hal_LockMemory: get_user_pages failed (%d)\n", result);
@@ -924,7 +928,11 @@ INTERNAL donttag void hal_UnlockMemory(v
         if (alenlist->flags & PS_ALENLIST_READ) {
           SetPageDirty(pages[i]);
         }
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,6,0)
         page_cache_release(pages[i]);
+#else
+        put_page(pages[i]);
+#endif
       }
     }
 
